apps/codeforafrica/src/lib/api.netlify-cms/getGetInTouch.js

```diff
@@ -1,7 +1,9 @@
 import { getCollectionBySlug } from "./utils";
 
 export default function getGetInTouch() {
-  return getCollectionBySlug("content/pages", "about", ["get-in-touch"]).items[
-    "get-in-touch"
-  ];
+  const getInTouch = getCollectionBySlug("content/pages", "about", [
+    "get-in-touch",
+  ]).items["get-in-touch"];
+
+  return { ...getInTouch, slug: "get-in-touch" };
```

Comment:
No need to do it this way. `getCollectionBySlug` already [supports](https://github.com/CodeForAfrica/ui/blob/ed09ce7b7ab86baa7a12b8b427d794cb6d188ccc/apps/codeforafrica/src/lib/api.netlify-cms/utils.js#L25-L26) `slug` as one of the fields